index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
+    <div class="calculator-row">
+      <div class="calculator-cell calculator-AC">AC</div>
```

Comment:
Why can't I just use divs instead of buttons? Is it just because the purpose of a button is to be clicked so why have a div do a button's work?